ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/Wizard/ScanConfigWizardForm.tsx

```diff
@@ -1,12 +1,7 @@
 import React, { ReactElement, useCallback, useRef, useState } from 'react';
 import { useNavigate } from 'react-router-dom';
-import { Button, Modal } from '@patternfly/react-core';
-import {
-    Wizard,
-    WizardContextConsumer,
-    WizardFooter,
-    WizardStep,
-} from '@patternfly/react-core/deprecated';
+import { Button, Modal, Wizard, WizardStep } from '@patternfly/react-core';
+import type { WizardStepType } from '@patternfly/react-core';
```

Comment:
Just curious, is separate `import type` statement a conscious decision, or eager addition by IDE?

Reply:
It was an eager addition by IDE that I consciously left in. I like the additional clarity, and apparently it can help speed up compilation due to being fully erased at runtime.

It has a side-effect of not triggering module level side-effects, which although not important here, could be useful in some cases.